test/uk/gov/hmrc/merchandiseinbaggage/stubs/MibBackendStub.scala

```diff
@@ -123,7 +124,7 @@ object MibBackendStub extends MibConfiguration with CoreTestData {
   )(implicit server: WireMockServer): StubMapping =
     server
       .stubFor(
-        post(urlPathEqualTo(s"$amendsPlusExistingCalculationsUrl"))
+        post(urlPathEqualTo(s"${config.mibAmendsPlusExistingCalculationsUrl}"))
```

Comment:
This can just be `post(urlPathEqualTo("/declare-commercial-goods/amend-calculations"))`